UniTimetable.php

```diff
@@ -154,36 +141,36 @@ classroomID smallint UNSIGNED NOT NULL COMMENT 'Fkey from classrooms',
     dbDelta($sql);
     
     //create view
-    $wpdb->query("CREATE  OR REPLACE VIEW $lecturesView AS
-            SELECT
-                periodID,
-                lectureID,
-                semester,
-                $lecturesTable.groupID,
-                $lecturesTable.classroomID,
-                $lecturesTable.teacherID,
-                start,
-                end,
-                groupName,
-                $subjectsTable.subjectID,
-                $subjectsTable.title AS subjectTitle,
-                $subjectsTable.type AS subjectType,
-                color,
-                $classroomsTable.name AS classroomName,
-                $classroomsTable.type AS classroomType,
-                surname as teacherSurname,
-                $teachersTable.name as teacherName
-            FROM
-                $lecturesTable,
-                $groupsTable,
-                $subjectsTable,
-                $classroomsTable,
-                $teachersTable
-            WHERE
-                $lecturesTable.groupID = $groupsTable.groupID
-                    AND $groupsTable.subjectID = $subjectsTable.subjectID
-                    AND $lecturesTable.classroomID = $classroomsTable.classroomID
-                    AND $lecturesTable.teacherID = $teachersTable.teacherID;");
+    //Ignore for now, our schema and actual scheduling storage is not fixed

```

Comment:
The lectures view depends on a lot of tables and until we have settled down on one schema, it needs to be disabled